### ISSUE TITLE
Error loading vcf with some SV types

### ISSUE THREAD

**Issue by cboursnell** [PapenfussLab/StructuralVariantAnnotation#26]
I have a GRIDSS vcf that I have updated the svtype field (which was all BND to start with) to be taken from the output of Clove. So these are BND, DEL, INV, TAN, TRA. 
When I try to load this vcf into R and get the breakpointRanges I get an error.
My R is:
```
vcf <- readVcf(input, "hg38")
gr <- breakpointRanges(vcf)
```
and the error is 
```
Error in names(cgr)[is.na(end), ] : incorrect number of dimensions
```
Update: I have tried just changing the deletion events from BND to DEL and this gives the same error. I can provide a minimal reproducible example if needed.

**Comment by d-cameron** [comment]
Yes, please provide a minimal reproduction VCF with just one or two of the offending records.
